requirements.txt

```
@@ -0,0 +1 @@
+git+https://github.com/fkie-cad/Codescanner.git
```

Should we pin the commit? 8)

```suggestion
git+https://github.com/fkie-cad/Codescanner.git@02262b6345dfb156ddcc822f4e84586a7c6c1a7a
```